org.osgi.service.jdbc/src/org/osgi/service/jdbc/DataSourceFactory.java

```diff
@@ -40,6 +40,56 @@
  * @ThreadSafe
  */
 public interface DataSourceFactory {
+	/**
+	 * Service property used by a {@code DataSourceFactory} to declare the
+	 * capabilities when registering a JDBC DataSourceFactory service. Clients
+	 * may filter or test this property to determine if the comparison method
+	 * could be used.
+	 * 
+	 * @since 1.1.0
```

Comment:
Just `1.1`. Not `1.1.0` as we would never change API on a micro version change. So all new API are added on a major or minor change. You will see that we only use major.minor in all other `@since` java tags.